runtime/port/unix/j9sysinfo.c

```diff
@@ -479,20 +479,34 @@ getLinuxPPCDescription(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 
 	/* initialize auxv prior to querying the auxv */
 	if (prefetch_auxv()) {
+_error:
 		desc->processor = PROCESSOR_PPC_UNKNOWN;
 		desc->physicalProcessor = PROCESSOR_PPC_UNKNOWN;
+		desc->features[0] = 0;
+		desc->features[1] = 0;
 		return -1;
 	}
 
 	/* Linux PPC processor */
 	platform = (char *) query_auxv(AT_PLATFORM);
+	if ((NULL == platform) || (((char *) -1) == platform)) {
+		goto _error;
```

Comment:
This goto is grotesque.  Why not initialize outside the info and avoid it all together?

Reply:
:)  goto is used this way throughout the VM code base to simplify error handling.  

If you prefer the code be written another way, please open a PR.